controllers/storagecluster/cephcluster.go

```diff
@@ -654,6 +654,12 @@ func newCephCluster(sc *ocsv1.StorageCluster, cephImage string, kmsConfigMap *co
 	cephCluster.Spec.Security.KeyRotation.Enabled = isEnabled
 	cephCluster.Spec.Security.KeyRotation.Schedule = rotationSchedule
 
+	if sc.Spec.ManagedResources.CephCluster.WipeDevicesFromOtherClusters {
+		cephCluster.Spec.CleanupPolicy = rookCephv1.CleanupPolicySpec{
+			WipeDevicesFromOtherClusters: sc.Spec.ManagedResources.CephCluster.WipeDevicesFromOtherClusters,
```

Comment:
As suggested before it might be better to expose & pass the whole CleanUpPolicy.